pkg/virt-controller/services/multus_annotations.go

```diff
@@ -82,18 +75,18 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 	return "", nil
 }
 
-func newMultusAnnotationData(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string) multusNetworkAnnotation {
+func newMultusAnnotationData(namespace string, interfaces []v1.Interface, network v1.Network, podInterfaceName string) networkv1.NetworkSelectionElement {
 	multusIface := vmispec.LookupInterfaceByName(interfaces, network.Name)
 	namespace, networkName := getNamespaceAndNetworkName(namespace, network.Multus.NetworkName)
 	var multusIfaceMac string
 	if multusIface != nil {
 		multusIfaceMac = multusIface.MacAddress
 	}
-	return multusNetworkAnnotation{
-		InterfaceName: podInterfaceName,
-		Mac:           multusIfaceMac,
-		Namespace:     namespace,
-		NetworkName:   networkName,
+	return networkv1.NetworkSelectionElement{
+		InterfaceRequest: podInterfaceName,
+		MacRequest:       multusIfaceMac,
+		Namespace:        namespace,
+		Name:             networkName,
 	}
```

Comment:
nit: Please consider initializing the fields in their new order:
```suggestion
	return networkv1.NetworkSelectionElement{
		Name:             networkName,
		Namespace:        namespace,
		MacRequest:       multusIfaceMac,
       		InterfaceRequest: podInterfaceName,
	}
```

It could potentially help future reviewers understand why the tests were changed.

Reply:
I don't follow how changing the order of the struct definition will help reviewers understand the reason why the test order has changed.

IMHO that would even be counter-productive - i.e. hint the key order when instantiating the struct matters to the deserializer. 

While I won't pretend I know, I don't think it matters.

But I won't insist; if you really want me to re-order this, I can comply.

Reply:
I won't insist.
This is just a nit.

Thank you!